subsys/bluetooth/host/hci_core.c

```diff
@@ -21,12 +21,14 @@
 
 #define BT_DBG_ENABLED IS_ENABLED(CONFIG_BLUETOOTH_DEBUG_HCI_CORE)
 #include <bluetooth/log.h>
+#include <common/log.h>
```

Comment:
I didn't do that because they are not supposed to be public, since they cannot be called from preemptable threads